CMakeLists.txt

```diff
@@ -98,7 +96,7 @@ list(APPEND HTS_SOURCES
 
 add_subdirectory(lib/libhts)
 
-set(DEPLIBS ${p8-platform_LIBRARIES} hts)
+set(DEPLIBS hts)
 if(WIN32)
   list(APPEND DEPLIBS ws2_32)
```

Comment:
This also no more needed as the `ASIO` makes `#   pragma comment(lib, "ws2_32.lib")`

Reply:
Okay, will remove, then.